pilot/cmd/pilot-discovery/app/cmd.go

```diff
@@ -127,7 +127,7 @@ func addFlags(c *cobra.Command) {
 		serverArgs.RegistryOptions.ClusterRegistriesNamespace, "Namespace for ConfigMap which stores clusters configs")
 	c.PersistentFlags().StringVar(&serverArgs.RegistryOptions.KubeConfig, "kubeconfig", "",
 		"Use a Kubernetes configuration file instead of in-cluster configuration")
-	c.PersistentFlags().StringVar(&serverArgs.MeshConfigFile, "meshConfig", "./etc/istio/config/mesh",
+	c.PersistentFlags().StringVar(&serverArgs.MeshConfigFile, "meshConfig", "/etc/istio/config/mesh",
```

Comment:
I think @costinm added it intentionally. Any issue why you want to change?

Reply:
It may be easy to mislead.
Or adjust the default configuration path of networksConfig to a relative path `./etc/istio/config/meshNetworks`?

Reply:
looks reasonable

Reply:
As mentioned in the other comment: "This is intended to allow running without access to the root filesystem. In the standard docker images we run with WD=/ so it's the same in most cases"

We can make the mesh networks one a relative path too if we want, but shouldn't change the meshconfig path.

Reply:
ok, adjust the default configuration path of networksConfig to a relative path `./etc/istio/config/meshNetworks`.